modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/SwapEstimate.swift

```diff
@@ -36,7 +36,6 @@ public struct SwapEstimate {
     }
 
     public struct SwapStep {
-        public let unsignedSwapTransaction: UnsignedSwapTransaction
```

Comment:
We still use it in `SwapQuote`. Is it the same shape?

Reply:
yes, i checked `unsignedSwapTransaction: UnsignedSwapTransaction` and it looks unused, maybe has left after some migration.